runtime/src/lib.rs

```diff
@@ -945,25 +954,41 @@ construct_runtime! {
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Call, Event<T>, Origin} = 52,
 		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 53,
 
-		// ChainBridge
+		// Litentry pallets
 		ChainBridge: pallet_bridge::{Pallet, Call, Storage, Event<T>} = 60,
 		BridgeTransfer: pallet_bridge_transfer::{Pallet, Call, Event<T>, Storage} = 61,
-		// Litentry pallets
 		Drop3: pallet_drop3::{Pallet, Call, Storage, Event<T>} = 62,
+		ExtrinsicFilter: pallet_extrinsic_filter::{Pallet, Call, Storage, Event<T>} = 63,
 
 		// TMP
 		Sudo: pallet_sudo::{Pallet, Call, Storage, Config<T>, Event<T>} = 255,
 	}
 }
 
-pub struct BaseCallFilter;
-impl Contains<Call> for BaseCallFilter {
+pub struct SafeModeFilter;
+impl Contains<Call> for SafeModeFilter {
+	fn contains(call: &Call) -> bool {
+		matches!(
+			call,
+			Call::Sudo(_) |
+			// System
+			Call::System(_) | Call::Timestamp(_) | Call::ParachainSystem(_) |
+			// ExtrinsicFilter
+			Call::ExtrinsicFilter(_)
+		)
+	}
+}
+
+pub struct NormalModeFilter;
```

Comment:
If I understand it correctly, we set the current NormalModeFilter the same as SafeModeFilter, and modify this when we gradually add more functionalities here, but the SafeModeFilter stays unchanged. Am I right?

Reply:
> If I understand it correctly, we set the current NormalModeFilter the same as SafeModeFilter, and modify this when we gradually add more functionalities here, but the SafeModeFilter stays unchanged. Am I right?

That's correct.
`SafeModeFilter` contains only the minimum core calls which maintain the parachains running.

As we activate more pallets, `NormalModeFilter` will be augmented to contain those calls.